templates/base.html

```diff
@@ -13,6 +13,7 @@
 {%- if not language_strings -%}
     {%- set language_strings = load_data(path="themes/tabi/i18n/" ~ lang ~ ".toml", required=false) -%}
 {%- endif -%}
+{% set_global rtl_langs = ["ar","arc","az","dv","ff","he","ku","nqo","fa","rhg","syc","ur"] %}
```

Comment:
I think `set rtl_langs` is enough. `set_global` is useful to set a variable within a loop to be available outside the loop.